<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkyWindow</class>
 <widget class="QMainWindow" name="SkyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkyWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="leUserName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbUserName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbAPIKey">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Api Key:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAPIKey">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="lnLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>921</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConnectButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>20</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAPIUrl">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>631</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbRestAction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GET</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>POST</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="lnRestAction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>891</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="twHeaderValues">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>831</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="rowCount">
     <number>25</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
   </widget>
   <widget class="Line" name="lnHeaders">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>891</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>851</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>831</width>
      <height>192</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">2</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSkyBox">
    <property name="title">
     <string>SkyBox</string>
    </property>
   </widget>
   <addaction name="menuSkyBox"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
